cli/flox-rust-sdk/src/models/manifest/typed.rs

```diff
@@ -1030,10 +1062,7 @@ pub mod test {
         let expected = indoc! {r#"
             version = 1
 
-            [options.allow]
-            licenses = []
-
-            [options.semver]
+            [options]
```

Comment:
(could be follow-up) I don't think we shouldn't be serializing this or rendering it in `flox list -c` if it wasn't originally specified by the user.